src/Surfnet/AzureMfa/Infrastructure/Validator/Constraint/EmailDomainInConfiguration.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\AzureMfa\Infrastructure\Validator\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use function get_class;
+
+/**
+ * @Annotation
+ */
+class EmailDomainInConfiguration extends Constraint
+{
+    public $message = 'The provided email address did not match any of our configured email domains.';
+
+    public function validatedBy()
+    {
+        return get_class($this).'Validator';
```

Comment:
Please use `EmailDomainInConfigurationValidator::class` this will help to resolve usages in the future.
Is this going to work, no return type while declaring strict types?

Reply:
Good suggestion!